app/controllers/devise_token_auth/concerns/set_user_by_token.rb

```diff
@@ -123,11 +148,30 @@ def refresh_headers
       # cleared by sign out in the meantime
       return if @used_auth_by_token && @resource.tokens[@token.client].nil?
 
+      _auth_header_from_batch_request = auth_header_from_batch_request
+
       # update the response header
-      response.headers.merge!(auth_header_from_batch_request)
+      response.headers.merge!(_auth_header_from_batch_request)
+
+      # set a server cookie if configured
+      if DeviseTokenAuth.cookie_enabled
+        set_cookie(_auth_header_from_batch_request)
+      end
     end # end lock
   end
 
+  def set_cookie(auth_header)
+    cookie_attributes = DeviseTokenAuth.cookie_attributes
+    cookie_attributes.merge!(value: auth_header.to_json)
+
+    cookie_domain = get_cookie_domain
+    if cookie_domain.present?
+      cookie_attributes.merge!(domain: cookie_domain)
```

Comment:
This line is unreached by specs but it could be ok

Reply:
This conditional will be moot now, see [this recent commit](https://github.com/lynndylanhurley/devise_token_auth/pull/1453/commits/536340b3e1861fef0e595158b3588b747d765898).